libraries/icms/file/MediaUploadHandler.php

```diff
@@ -526,7 +526,7 @@ public function upload($chmod = 0644) {
 		if (!self::checkMaxHeight()) {
 			return false;
 		}
-		if (count($this->errors) > 0) {
+		if (count($errors) > 0) {
```

Comment:
Scrutinizer says that here could be some problems. So, I'm not sure if this is a goo fix.

Reply:
I looked at scrutinizer, and it look strange: they say that the private array $errors is not used, and after that, they say that the array is not defined. Perhaps there is a better way of handling this though (exceptions)